nirc_ehr/src/org/labkey/nirc_ehr/security/EHRVetTechPermission.java

```diff
@@ -0,0 +1,11 @@
+package org.labkey.nirc_ehr.security;
+
+import org.labkey.api.ehr.security.AbstractEHRPermission;
+
+public class EHRVetTechPermission extends AbstractEHRPermission
```

Comment:
Could it better to name this as NIRCEHRVetTechPermission to avoid confusion between permissions defined in ehrModules and center specific ones?

Reply:
Good call, thought I had named it with NIRC prefix, perhaps just in my head :)